capstone-backend/src/main/java/com/google/sps/servlets/Utility.java

```diff
@@ -52,15 +55,18 @@ public static JsonObject createRequestBodyJson(HttpServletRequest request) throw
   }
 
   /**
-   * Query the given collection by id. 
+   * Query the given collection by id.
+   * 
    * @param {collectionReference} reference to the appropriate database collection
-   * @param {request} request sent to the backend
-   * @param {response} response returned from the call
-   * @param {genericClass} a Generic class, used in casting documents from the database
-   * @return List<T> a singleton List of the Object that matches the ID in the request.
+   * @param {request}             request sent to the backend
+   * @param {response}            response returned from the call
+   * @param {genericClass}        a Generic class, used in casting documents from
+   *                              the database
+   * @return List<T> a singleton List of the Object that matches the ID in the
+   *         request.
    */
-  private static <T extends BaseEntity> List<T> getById(CollectionReference collectionReference, HttpServletRequest request, 
```

Comment:
Not sure if these style updates should be in this PR, but it can't hurt.

Reply:
Yeah that Gitpod formatting the code. I didn't really indent for it.

Reply:
Sounds good!

Reply:
Just want to try to avoid merge conflicts where possible, but none of these changes are too large so should be fairly easy to resolve.